pyNTCIREVAL/metrics/recall.py

```diff
@@ -0,0 +1,11 @@
+from .metric import Metric
+
+class Recall(Metric):
+    def __init__(self, total_positive):
+        self.total_positives = total_positive
+
+    def compute(self, labeled_ranked_list):
+        true_positives = sum(item[1] if item[1] is not None else 0 for item in labeled_ranked_list)
```

Comment:
To clarify what `item[1]` is, better to write the for-loop as follows:
`sum(grade if grade is not None else 0 for docid, grade in labeled_ranked_list)`

In addition, `grade` can be 1 or larger (e.g., 2). Thus, 
`len(grade for docid, grade in labeled_ranked_list if grade is not None and grade > 0)`